ofrak_core/ofrak/core/strings.py

```diff
@@ -145,13 +144,12 @@ async def unpack(self, resource: Resource, config: None) -> None:
         else:
             pattern = self.SHORT_STRING_PATTERN
 
-        data = await resource.get_data()
-
         children = [
             resource.create_child_from_view(
-                AsciiString(m.group(1).decode("ascii")), data_range=Range(m.start(), m.end())
+                AsciiString(string[:-1].decode("ascii")),
```

Comment:
Is the actual goal here to strip off the last byte? Or is it to `rstrip` at most one trailing null bytes? Might make more sense to implement as the latter, since that's closer to the actual intention.

Reply:
The regex only matches exactly one trailing null byte, but I agree that a `strip()` call is clearer.